addons/mail/static/src/models/autocomplete_input_view.js

```diff
@@ -51,6 +51,23 @@ registerModel({
             }
             return clear();
         },
+        /**
+         * @private
+         * @returns {string|FieldCommand}
+         */
+        _computeCustomClass() {
+            if (this.discussSidebarCategoryOwnerAsAddingItem) {
+                if (this.discussSidebarCategoryOwnerAsAddingItem === this.messaging.discuss.categoryChannel) {
+                    return 'o_DiscussSidebarCategory_newChannelAutocompleteSuggestions';
+                } else {
+                    return clear();
+                }
```

Comment:
This clear is actually useless in regard to the one at the end of the method

Reply:
Change done